src/compiler/transformers/es2015.ts

```diff
@@ -523,7 +523,7 @@ export function transformES2015(context: TransformationContext): (x: SourceFile
      * They are persisted between each SourceFile transformation and should not
      * be reset.
      */
-    let enabledSubstitutions: ES2015SubstitutionFlags;
+    let enabledSubstitutions!: ES2015SubstitutionFlags;

```

Comment:
It feels like all of these should just be initialized with `0` or some equivalent.

Reply:
CC @rbuckton

Reply:
Added a `None` flag